scripts/utils/starknet.py

```diff
@@ -238,13 +246,26 @@ def get_tx_url(tx_hash: int) -> str:
     return f"{NETWORK['explorer_url']}/tx/0x{tx_hash:064x}"
 
 
+def is_fixture_contract(contract_name):
+    return CONTRACTS_FIXTURES.get(contract_name) is not None
+
+
 def compile_contract(contract):
+    is_fixture = is_fixture_contract(contract["contract_name"])
+    contract_build_path = (
```

Comment:
could you call `get_artifact(contract['contract_name'])` here?

Reply:
makes sense, doing the refactor.